x/precisebank/keeper/view_test.go

```diff
@@ -0,0 +1,109 @@
+package keeper_test
+
+import (
+	"testing"
+
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/kava-labs/kava/x/precisebank/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestKeeper_GetBalance(t *testing.T) {
+	tests := []struct {
```

Comment:
can you add a test case for zero ukava balance (zero &/or nil) with nonzero akava balance?
(ie, i send a never-used-before account <1e12 akava then query that account's balance)